Makefile

```
@@ -16,6 +16,7 @@ install:
 	install -m755 modules-load ${DESTDIR}/${PREFIX}/sbin/modules-load
 	install -m755 seedrng ${DESTDIR}/${PREFIX}/sbin/seedrng
 	install -m755 zzz ${DESTDIR}/${PREFIX}/sbin
+	install -m755 runit-analyze ${DESTDIR}/${PREFIX}/sbin
```

this should also be renamed
```suggestion
	install -m755 runit-startuptime ${DESTDIR}/${PREFIX}/sbin
```

---

also maybe the manpage should be installed?